style.css

```diff
@@ -0,0 +1,2456 @@
+  width: 19.5rem;
+}
+
+.container-basic-8LZ {
+  background-color: #fff;
+  border: solid 0.1rem #cfd8dc;
+  border-radius: 0.2rem;
+  box-sizing: border-box;
+  color: #b3bac5;
+  flex-shrink: 0;
+  font-family: "Poppins", sans-serif;
+  font-size: 1.5rem;
+  font-weight: 400;
+  height: 17rem;
+  line-height: 1.6;
+  padding: 1.5rem 1.6rem;
+  white-space: nowrap;
+  width: 100%;
+}
+
+.normal-button-primary-enabled-bE9 {
+  align-items: center;
+  background-color: #fff;
+  border-radius: 0.8rem;
+  color: #6070ff;
+  display: flex;
+  flex-shrink: 0;
+  font-family: "Poppins", sans-serif;
+  font-size: 1.7rem;
+  font-weight: 500;
+  height: 4.8rem;
+  justify-content: center;
+  letter-spacing: 0.051rem;
+  line-height: 1.4117647059;
+  margin: 0 15.25rem;
+  overflow: hidden;
+  position: relative;
+  white-space: nowrap;
+  width: calc(100% - 30.5rem);
+}
+
+.aboutmyself-left-block {
+  box-sizing: border-box;
+  height: 45.2rem;
+  left: 14.2rem;
+  overflow: hidden;
+  padding-top: 0.5rem;
+  position: absolute;
+  top: 14.2rem;
+  width: 44.7rem;
+}
+
+.abouttext {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2.4rem;
+  overflow: hidden;
+  position: relative;
+}
+
+.primary-text-Deq {
+  color: #091e42;
+  display: flex;
+  flex-shrink: 0;
+  font-family: "Poppins", sans-serif;
+  font-size: 4rem;
+  font-weight: 700;
+  line-height: 1.3;
+  margin-bottom: 1.7rem;
+  max-width: 13.6rem;
+}
+
+.primary-text {
+  color: #344563;
+  flex-shrink: 0;
+  font-family: "Poppins", sans-serif;
+  font-size: 1.6rem;
+  font-weight: 400;
+  line-height: 1.5;
+  margin-bottom: 1.8rem;
+  max-width: 44.5rem;
+}
+
+.aboutsocial-media {
+  flex-shrink: 1;
+  overflow: hidden;
+  position: relative;
+  width: 20rem;
+}
+
+.frame-1 {
+  align-items: center;
+  display: flex;
+  overflow: hidden;
+  position: static;
+  width: 100%;
+}
+
+.social-media-image {
+  height: 40px;
+  width: 40px;
+  border-radius: 8px;
+  padding: 8px;
+}
+
+.skill-list-rWm {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 36rem;
+  left: 61.4rem;
+  overflow: hidden;
+  position: absolute;
+  row-gap: 1.6rem;
+  top: 14.2rem;
+  width: 68.4rem;
+}
+
+.frame-12-xJu {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  overflow: hidden;
+  padding: 1.6rem 0 1.2rem 0;
+  position: relative;
+  width: 100%;
+}
+
+.frame-7-tTT {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  margin-bottom: 1.2rem;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+}
+
+.auto-group-bdym-qdb {
+  align-items: center;
+  display: flex;
+  flex-shrink: 0;
+  margin: 0 2.6rem 1.5rem 2.4rem;
+  width: calc(100% - 5rem);
+}
+
+/* .languages {
+  color: #344563;
+  display: flex;
+  flex-shrink: 0;
+  font-family: "Poppins", sans-serif;
+  font-size: 2rem;
+  font-weight: 500;
+  line-height: 1.2;
+  margin-right: 48.1rem;
+  white-space: nowrap;
+} */
+
```

Comment:
- [ ] Kindly remove this commented piece of code, as it hurts the overall organization and professionalism of your code.

Reply:
fixed